Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/BlendGraphWidget.cpp

```diff
@@ -1282,6 +1245,100 @@ namespace EMStudio
         }
     }
 
+    void BlendGraphWidget::CreateNodeGroup()
+    {
+        AZ_Assert(sender()->inherits("QAction"), "CreateNodeGroup called apart from a connection to a QAction's signal");
+
+        // Don't create node group if there's no selection (the create group option should only appear when right clicking a node)
+        const QItemSelection selection = m_plugin->GetAnimGraphModel().GetSelectionModel().selection();
+        const QModelIndexList selectionList = selection.indexes();
+        if (selectionList.empty())
+        {
+            return;
+        }
+
+        // add the parameter
+        EMotionFX::AnimGraph* animGraph = m_plugin->GetActiveAnimGraph();
+        if (animGraph == nullptr)
+        {
+            MCore::LogWarning("BlendGraphWidget::OnNodeGroupCreated() - No AnimGraph active!");
+            return;
+        }
+
+        AZStd::string commandString;
+        AZStd::string resultString;
+        commandString = AZStd::string::format("AnimGraphAddNodeGroup -animGraphID %i", animGraph->GetID());
+
```

Comment:
suggestion: Could we update this to...

```
const AZStd::string commandString; = AZStd::string::format("AnimGraphAddNodeGroup -animGraphID %i", animGraph->GetID());

AZStd::string resultString;
if (GetCommandManager()->ExecuteCommand(commandString.c_str(), resultString) == false)
```

So `resultString` is closer to its first use and we avoid the default construction of `commandString` (and can also make it `const` 🙂)